bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/TaskBar.java

```diff
@@ -65,6 +65,7 @@ public class TaskBar extends Widget {
 
 TaskBar (Display display, int style) {
 	this.display = display;
+	this.nativeZoom = display.getDeviceZoom();
```

Comment:
```suggestion
	this.nativeZoom = display.getNativeDeviceZoom();
```

Reply:
See https://github.com/eclipse-platform/eclipse.platform.swt/pull/1268#discussion_r1658595251

Reply:
My comment was invalid (see https://github.com/eclipse-platform/eclipse.platform.swt/pull/1268#discussion_r1660574063).

Reply:
So no need to change anything here, right @HeikoKlare ?

Reply:
Not at this place, thus I had resolved this conversation. But why did you resolve my other comment? That one requires some adaptation.

Reply:
> ... But why did you resolve my other comment? That one requires some adaptation.

Sloppy fingers, sorry!